mava/configs/system/sac/ff_isac.yaml

```diff
@@ -0,0 +1,35 @@
+# --- Defaults FF-ISAC ---
+seed: 1
+
+# --- Agent observations ---
+add_agent_id: False
+
+# --- RL hyperparameters ---
+# step related
+total_timesteps: 2e7 # Set the total environment steps.
+# If unspecified, it's derived from num_updates; otherwise, num_updates adjusts based on this value.
+num_updates: 625000 # Number of updates
```

Comment:
If we want to use a **large** `num_updates`, shouldn't we ignore the `total_timesteps` setting? Or just set one of them 🤔